homeassistant/components/aqualogic/__init__.py

```diff
@@ -25,7 +27,12 @@
 CONFIG_SCHEMA = vol.Schema(
     {
         DOMAIN: vol.Schema(
-            {vol.Required(CONF_HOST): cv.string, vol.Required(CONF_PORT): cv.port}
+            {
+                vol.Optional(CONF_DEVICE, default="socket"): cv.string,
```

Comment:
Changes to the YAML configuration that communicates with a device or service is no longer accepted.

This integration needs to be converted to use the configuration flow first.

For more information see: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on implementing the configuration flow, please consult or developer documentation.

Reply:
Given this is a change that is being forced on existing integrations, I would have expected better documentation. A migration guide would be helpful.

Reply:
Thanks, not sure how to answer to that response. If you need help, we are all happy to provide that. What part of the guides don't you understand? Which issue did you run into?

I would highly recommend on joining or Discord chat as we can keep the conversation shorter and quicker.

👍

Reply:
i notice the update included serial port support...how would i use the USB RS485 serial port on my raspi to connect? I only see how to use the Ethernet IP address and port like below..
aqualogic:
  host: xx.xx.xx.xx
  port: xxxx